src/data.h

```diff
@@ -58,6 +64,12 @@ struct PFNeutralObj {
 	etaphi_t hwEta, hwPhi; // relative to the region center, at calo
 	particleid_t hwId;
 };
+struct PFMuonObj {
+	pt_t hwPt;
+	etaphi_t hwEta, hwPhi; // relative to the region center, at vtx(?)
+	particleid_t hwId;
+	z0_t hwZ0;
+};
```

Comment:
for the moment I would stay with the same object type, it will allow more code re-use.